lms/djangoapps/lti_provider/views.py

```diff
@@ -70,11 +83,13 @@ def lti_launch(request, course_id, usage_id):
             params['oauth_consumer_key']
         )
     except LtiConsumer.DoesNotExist:
-        return HttpResponseForbidden()
+        template403 = Template(render_to_string('static_templates/403.html', {'return_url': return_url}))
+        return HttpResponseForbidden(template403.render())
```

Comment:
What is the difference between this feature and edx built-in? only in "return-back-url" ? do we need some user friendly message or something like this?